pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -233,7 +233,7 @@ func Convert_v1_Disk_To_api_Disk(c *ConverterContext, diskDevice *v1.Disk, disk
 		}
 		volumeStatus, ok := volumeStatusMap[diskDevice.Name]
 		if ok && volumeStatus.PersistentVolumeClaimInfo != nil {
-			disk.FilesystemOverhead = volumeStatus.PersistentVolumeClaimInfo.FilesystemOverhead
+			disk.FilesystemOverhead = (*string)(volumeStatus.PersistentVolumeClaimInfo.FilesystemOverhead)
```

Comment:
Why don't we change the API as well?

Reply:
Done :+1: 

I also created a Percent type alias to keep the same type safety while dropping the dependency.